verl/trainer/config/sft_trainer.yaml

```diff
@@ -23,6 +23,7 @@ model:
   lora_rank: 0  # Set to positive value to enable LoRA (e.g., 32)
   lora_alpha: 16  # LoRA scaling factor
   target_modules: all-linear  # Target modules for LoRA adaptation
+  use_liger: True
```

Comment:
not sure if we want to enable it by default. thoughts? @PeterSH6 @vermouth1992

Reply:
Reverted to default false!

Reply:
Thanks! It would be great to mention this option in https://github.com/volcengine/verl/blob/main/docs/perf/perf_tuning.rst as well~

Reply:
OpenHands helped me did it here: https://github.com/volcengine/verl/pull/178 :)